params/config.go

```diff
@@ -106,7 +109,10 @@ var (
 		ConstantinopleBlock: big.NewInt(4230000),
 		PetersburgBlock:     big.NewInt(4939394),
 		IstanbulBlock:       big.NewInt(6485846),
-		Ethash:              new(EthashConfig),
+		Istanbul: &IstanbulConfig{
+			Epoch:          30000,
```

Comment:
Nit: If it doesn't break anything, 17280 would be the more canonical default value.
```suggestion
			Epoch:          17280,
```